src/components/Modal/Modal.css

```diff
@@ -12,35 +12,47 @@
 }
 
 .close-modal {
-  color: black;
   align-self: flex-start;
   border: 1px solid black;
-  padding: 0.5em;
+  padding: 0.25em;
+  margin-top: 1em;
 }
 
 .close-modal:hover {
   cursor: pointer;
   background-color: whitesmoke;
+  transform: scale(1.1);
 }
 
 .modal-content {
   margin: auto;
   background-color: white;
-  padding: 4em;
   border: 0.25em solid black;
-  height: 60vh;
-  width: 50vw;
```

Comment:
Hard to comment on css but I think we might want to add back in this height/width. I'm not super concerned with what it looks like at the moment, since we still have a lot of functionality and testing to work with, but will def need some media queries as well

Reply:
Do you mind if we leave this for now - and make changes when there's a more concerted CSS effort?